<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ProxyDialog</class>
    <widget class="QDialog" name="ProxyDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>200</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Proxy Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QRadioButton" name="radioNoProxy">
                    <property name="text">
                        <string>No Proxy</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QRadioButton" name="radioSystemProxy">
                    <property name="text">
                        <string>System Proxy</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QRadioButton" name="radioHTTP">
                    <property name="text">
                        <string>HTTP Proxy</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="widgetHTTP">
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                <item>
                                    <widget class="QLabel" name="labelHTTP">
                                        <property name="text">
                                            <string>Address:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="lineEditHTTP"/>
                                </item>
                                <item>
                                    <widget class="QLabel" name="labelHTTPPort">
                                        <property name="text">
                                            <string>Port:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QSpinBox" name="spinBoxHTTPPort">
                                        <property name="maximum">
                                            <number>65535</number>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="QCheckBox" name="checkBoxHTTPAuth">
                                <property name="text">
                                    <string>Use Authentication</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QWidget" name="widgetHTTPAuth">
                                <layout class="QHBoxLayout" name="horizontalLayout_4">
                                    <item>
                                        <widget class="QLabel" name="labelHTTPUsername">
                                            <property name="text">
                                                <string>Username:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="lineEditHTTPUsername"/>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="labelHTTPPassword">
                                            <property name="text">
                                                <string>Password:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="lineEditHTTPPassword">
                                            <property name="echoMode">
                                                <enum>QLineEdit::Password</enum>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QRadioButton" name="radioSocks5">
                    <property name="text">
                        <string>SOCKS5 Proxy</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QWidget" name="widgetSocks5">
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout_3">
                                <item>
                                    <widget class="QLabel" name="labelSocks5">
                                        <property name="text">
                                            <string>Address:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLineEdit" name="lineEditSocks5"/>
                                </item>
                                <item>
                                    <widget class="QLabel" name="labelSocks5Port">
                                        <property name="text">
                                            <string>Port:</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QSpinBox" name="spinBoxSocks5Port">
                                        <property name="maximum">
                                            <number>65535</number>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="QCheckBox" name="checkBoxSocks5Auth">
                                <property name="text">
                                    <string>Use Authentication</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QWidget" name="widgetSocks5Auth">
                                <layout class="QHBoxLayout" name="horizontalLayout_5">
                                    <item>
                                        <widget class="QLabel" name="labelSocks5Username">
                                            <property name="text">
                                                <string>Username:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="lineEditSocks5Username"/>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="labelSocks5Password">
                                            <property name="text">
                                                <string>Password:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="lineEditSocks5Password">
                                            <property name="echoMode">
                                                <enum>QLineEdit::Password</enum>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="buttonOK">
                    <property name="text">
                        <string>OK</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
